odxtools/flashdata.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import Any
+from xml.etree import ElementTree
+
+from .dataformat import Dataformat
+from .element import IdentifiableElement
+from .encryptcompressmethod import EncryptCompressMethod
+from .exceptions import odxrequire
+from .odxdoccontext import OdxDocContext
+from .odxlink import OdxLinkDatabase, OdxLinkId
+from .snrefcontext import SnRefContext
+from .utils import dataclass_fields_asdict
+
+
+@dataclass(kw_only=True)
+class Flashdata(IdentifiableElement):
+    size_length: int | None = None
+    address_length: int | None = None
+    dataformat: Dataformat
```

Comment:
```suggestion
    data_format: Dataformat
```

Reply:
I agree, but the spec calls the XML tag `DATAFORMAT`...

Reply:
it's a clear typo in the spec, not the first or last that we run into, mo need to follow it

Reply:
I'm not so sure: database is used as a single word in 99.5% of all cases...